python/lsst/obs/base/instrument_tests.py

```diff
@@ -104,14 +151,32 @@ def getRawFormatter(self, dataId):
         # Docstring inherited fromt Instrument.getRawFormatter.
         return DummyCamYamlWcsFormatter
 
-    def writeCuratedCalibrations(self, butler):
-        pass
-
     def applyConfigOverrides(self, name, config):
         pass
 
-    def makeDataIdTranslatorFactory(self) -> TranslatorFactory:
-        return TranslatorFactory()
+    def writeAdditionalCuratedCalibrations(
+        self, butler: Butler, collection: Optional[str] = None, labels: Sequence[str] = ()
+    ) -> None:
+        # We want to test the standard curated calibration ingest
+        # but we do not have a standard class to use. There is no way
+        # at the moment to inject a new class into the standard list
+        # so instead use the additional path.
```

Comment:
I don't understand this comment.  What is "the additional path"?

Reply:
This method: `writeAdditionalCuratedCalibrations` rather than using the "standard" list. Currently the standard list is a package constant and instrument subclasses can't add to that list. That's probably not great but I punted on fixing that for now.

Reply:
so maybe amend this to say "...into the standard list of curated calibrations that are a package constant.  Instead, we use ???" Because I don't know what "the additional path" refers to.  Code path? Filesystem path?